hydromt/workflows/forcing.py

```diff
@@ -414,6 +562,105 @@ def pet_makkink(temp, press, k_in, timestep=86400, cp=1005.0):
+    wind: xarray.DataArray,
+    temp_dew: xarray.DataArray,
+    dem: xarray.DataArray,
+    var: str = "temp_dew",
+) -> xarray.DataArray:
+    """
+    Estimate daily reference evapotranspiration (ETo) from a hypothetical
+    short grass reference surface using the FAO-56 Penman-Monteith equation.
+
+    Actual vapor pressure is derived either from relative humidity or dewpoint temperature (depending on var_for_avp_name).
+
+    Based on equation 6 in Allen et al (1998) and using the functions provided by the pyet package ()
+
+    Parameters
+    ----------
+    temp : xarray.DataArray
+        DataArray with daily temperature [°C]
+    temp_max : xarray.DataArray
+        DataArray with maximum daily temperature [°C]
+    temp_min : xarray.DataArray
+        DataArray with minimum daily temperature [°C]
+    press : xarray.DataArray
+        DataArray with pressure [hPa]
+    kin : xarray.DataArray
+        DataArray with global radiation [W m-2]
+    wind : xarray.DataArray
+        DataArray with wind speed at 2m above the surface [m s-1]
+    temp_dew : xarray.DataArray
+        DataArray with either temp_dew (dewpoint temperature at 2m above surface [°C]) or rh (relative humidity [%]) to estimate actual vapor pressure
+    dem : xarray.DataArray
+        DataArray with elevation at model resolution [m]
+    var : str, optional
+        String with variable name used to estimate actual vapor pressure (chose from ["temp_dew", "rh"])
+
+    Returns
+    -------
+    xarray.DataArray
+        DataArray with the estimated daily reference evapotranspiration pet [mm d-1]
+
+    Raises
+    ------
+    ModuleNotFoundError
+        In case the pyet module is not installed
+
+    """
+    # Small check for libraries
+    if not HAS_PYET:
+        raise ModuleNotFoundError("Penman-Monteith FAO-56 requires the 'pyet' library")
+
+    # Precalculated variables:
+    lat = kin[kin.raster.y_dim] * (np.pi / 180)  # latitude in radians
+
+    # Vapor pressure
+    svp = pyet.calc_e0(tmean=temp)
+
+    if var == "temp_dew":
+        avp = pyet.calc_e0(tmean=temp_dew)
+    elif var == "rh":
+        avp = pyet.calc_e0(tmax=temp_max, tmin=temp_min, rh=temp_dew)
+
+    # Net radiation
+    dates = pyet.utils.get_index(kin)
+    er = pyet.extraterrestrial_r(
+        dates, lat
+    )  # Extraterrestrial daily radiation [MJ m-2 d-1]
+    csr = pyet.calc_rso(er, dem)  # Clear-sky solar radiation [MJ m-2 day-1]
+
+    # Net shortwave radiation [MJ m-2 d-1]
+    swr = pyet.calc_rad_short(kin * (86400 / 1e6))
+
+    # Net longwave radiation [MJ m-2 d-1]
+    lwr = pyet.calc_rad_long(
+        kin * (86400 / 1e6), tmax=temp_max, tmin=temp_min, rso=csr, ea=avp
+    )
+    nr = swr - lwr  # daily net radiation
+
+    # Penman Monteith FAO-56
+    gamma = pyet.calc_psy(press)  # psychrometric constant
+    dlt = pyet.calc_vpc(
+        temp
+    )  # Slope of saturation vapour pressure curve at air Temperature [kPa °C-1].
+
+    gamma1 = gamma * (1 + 0.34 * wind)
```

Comment:
can we add more description about the variables / units used in this function?

Reply:
We probably should. I'll look at it